public/pages/CreateTrigger/containers/DefineTrigger/utils/validation.js

```diff
@@ -1,34 +1,35 @@
 /*
- *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *
- *   Licensed under the Apache License, Version 2.0 (the "License").
- *   You may not use this file except in compliance with the License.
- *   A copy of the License is located at
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
  *
- *       http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
  *
- *   or in the "license" file accompanying this file. This file is distributed
- *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- *   express or implied. See the License for the specific language governing
- *   permissions and limitations under the License.
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
  */
 
 import { FORMIK_INITIAL_TRIGGER_VALUES } from '../../CreateTrigger/utils/constants';
 
-export const validateTriggerName = (triggers, triggerToEdit) => (value) => {
+export const validateTriggerName = (triggers, triggerToEdit, fieldPath) => (value) => {
   if (!value) return 'Required';
   const nameExists = triggers.filter((trigger) => {
     const triggerId = _.get(
       trigger,
-      'aggregation_trigger.id',
+      `aggregation_trigger.id`,
```

Comment:
Any reason to change the single quotes to backticks?

Reply:
Ah, good catch! This may be a leftover change from some experimentation. I'll fix it when I submit the PR for the traditional trigger assets. Thank you for catching it!